### ISSUE TITLE
Transition from main menu to cinematic

### ISSUE THREAD

**Issue by AlexanderRavenheart** [AlexanderRavenheart/LDJam-45-SAR#14]
I think a fade to black from main menu and slow fade-in of the player pod with just breathing sounds for a bit would be a cool addition.

**Comment by StefanSava** [comment]
yep, it would probably be a improvement, also don't forget the skewed image.
